mcu-util/src/orb/mod.rs

```diff
@@ -84,6 +84,10 @@ impl Orb {
         }
     }
 
+    pub fn borrow_mut_main_board(&mut self) -> &mut MainBoard {
```

Comment:
could call it `main_board_mut(&mut self)` and `main_board(&self)`? This is the convention in rust.